sql/core/src/test/scala/org/apache/spark/sql/streaming/StreamingQuerySuite.scala

```diff
@@ -100,28 +110,138 @@ class StreamingQuerySuite extends StreamTest with BeforeAndAfter {
     )
   }
 
-  testQuietly("source and sink statuses") {
+  testQuietly("query statuses") {
```

Comment:
What do you mean? Various tests across the testsuites? 
I put it this way so that it corresponds to the class structured. 
- The test for internal logic of StreamMetrics is in StreamMetricsSuite.
- The test for the statuses returned through StreamingQueryListener is in StreamingQueryListenerSuite.
- The test for the StreamingQuery.status is in StreamingQuerySuite.